server/plugin.go

```diff
@@ -77,3 +77,21 @@ func (p *Plugin) handleGetAttributes(w http.ResponseWriter, r *http.Request) {
 }
 
 // See https://developers.mattermost.com/extend/plugins/server/reference/
+func contains(arr []string, str string, userGroups []*model.Group) bool {
```

Comment:
This function seems to have two different responsibilities that don't need to be grouped. I'd suggest splitting it into different functions that have specific names for what they're achieving

Reply:
Done! Split into two different functions